gov_uk_dashboards/components/plotly/context_box.py

```diff
@@ -0,0 +1,29 @@
+"""Context_box"""
+from dash import html
+
+
+def put_context_banner():
```

Comment:
does put need to be included as part of function definition?

Reply:
function renamed to `context_banner()`